Mathlib/Algebra/Order/Group/Abs.lean

```diff
@@ -342,7 +342,7 @@ variable [LinearOrderedCommGroup α] {a b : α}
   obtain ha | ha := le_or_lt 1 a <;> obtain hb | hb := le_or_lt 1 b
   · simp [ha, hb, mabs_of_one_le, one_le_mul ha hb]
   · exact (lt_irrefl (1 : α) <| ha.trans_lt <| hab.trans_lt hb).elim
-  any_goals simp [ha.le, hb.le, mabs_of_le_one, mul_le_one', mul_comm]
+  on_goal 2 => simp [ha.le, hb.le, mabs_of_le_one, mul_le_one', mul_comm]
```

Comment:
On its own merits (i.e., ignoring the multi-goal linter), I'm not sure if this change is an improvement.